rules/unix-shell.data

```diff
@@ -677,15 +677,19 @@ dev/stdout
 dev/tcp
 dev/udp
 dev/zero
+etc/alternatives/rmt
```

Comment:
```suggestion
etc/alternatives
```
would be too broad to use just `etc/alternatives`? Probably everything in there is an executable, and can be used to bypass things (like using `/etc/alternatives/javac` for example?)

Reply:
We already do in `lfi-os-files.data`.